service/test/_registry_test.js

```diff
@@ -930,30 +946,56 @@ describe('Registry', function() {
       });
     });
 
-    describe('#updateDevices', function(){
+    describe('#updateDevices', function () {
       /*Tests_SRS_NODE_IOTHUB_REGISTRY_06_025: [The `updateDevices` method shall throw `ReferenceError` if the `devices` argument is falsy.]*/
-      [undefined, null].forEach(function(badDevices) {
-        testDevicesFalsyArg('updateDevices', 'devices', badDevices, ReferenceError);
+      [undefined, null].forEach(function (badDevices) {
+        it("throw `ReferenceError` if the `devices` argument is falsy", function () {
+          assert.throws(() => {
+            var registry = new Registry({
+              host: 'host',
+              sharedAccessSignature: 'sas'
+            });
+            registry.updateDevices(badDevices, false, () => {});
+          }, ReferenceError)
+        });
       });
 
       /*Tests_SRS_NODE_IOTHUB_REGISTRY_06_012: [The `updateDevices` method shall throw `ArgumentError` if any elements of devices do NOT contain a `deviceId` property.]*/
-      [undefined, null, ''].forEach(function(badDeviceId) {
-        testDevicesFalsyArg('updateDevices', 'devices', [{deviceId: 'goodDevice'},{ deviceId: badDeviceId }], errors.ArgumentError);
+      [undefined, null, ''].forEach(function (badDeviceId) {
+        it("throws `ArgumentError` if any elements of devices do NOT contain a `deviceId` property", function () {
+          assert.throws(() => {
+            var registry = new Registry({
+              host: 'host',
+              sharedAccessSignature: 'sas'
+            });
+            registry.updateDevices([{
+              deviceId: 'goodDevice'
+            }, {
+              deviceId: badDeviceId
+            }], false, () => {});
+          }, errors.ArgumentError)
```

Comment:
Doing this really makes it difficult to compare changes.  The spaces added really make for a lot of noise.

Reply:
Yeah, I'm really sorry about that.

Reply:
Done.

---
In reply to: [222830730](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222830730) [](ancestors = 222830730)